src/Components/Modal.js

```diff
@@ -0,0 +1,198 @@
+              <Styled_Icon>
+                <Icon icon="logos:react" width="25" height="25" />
+                <span>Overview</span>
+              </Styled_Icon>
+            </Styled_Span>
+
+            <span>NAME: {content.name}</span>
+            <span>ATOMIC MASS: {content.atomic_mass}</span>
+            <span>NAMED BY: {content.named_by}</span>
+            <span>CATEGORY: {content.category}</span>
+            <span>DISCOVERED BY: {content.discovered_by}</span>
+            <span>APPEARANCE: {content.appearance}</span>
+
+            <Styled_Span>
+              <Styled_Icon>
+                <Icon
+                  icon="fontisto:laboratory"
+                  color="red"
+                  width="25"
+                  height="25"
+                />
+                <span>Properties</span>
+              </Styled_Icon>
+            </Styled_Span>
+
+            <StyledLine>
+              <span>BOILING POINT: {content.boil}</span>
+            </StyledLine>
+
+            {/* //Line not working */}
+
+            <span>MELTING POINT: {content.melt}</span>
+            <span>DENSITY: {content.density}</span>
+            <span>SOURCE: {content.source}</span>
+            {/* <span className="name">{content.molar_heat}</span> */}
+            <span>ATOMIC NUMBER: {content.number}</span>
+            <span>PERIOD: {content.period}</span>
+            <span>PHASE AT STP: {content.phase}</span>
+            <span>SUMMARY: {content.summary}</span>
+            <span>{content.shells}</span>
+            <span>
+              ELECTRONIC CONFIGURATION: {content.electron_configuration}
+            </span>
+            {/* <span className="name">{content.electron_configuration_semantic}</span> */}
+
+            <Styled_Span>
+              <Styled_Icon>
+                <Icon
+                  icon="jam:triangle-danger"
+                  color="red"
+                  width="25"
+                  height="25"
+                />
+                <span>Reactivity</span>
+              </Styled_Icon>
+            </Styled_Span>
+
+            <span>ELECTRON AFFINITY: {content.electron_affinity}</span>
+            <span>ELECTRONEGATIVITY: {content.electronegativity_pauling}</span>
+            <span>IONIZATION ENERGIES: {content.ionization_energies}</span>
+          </StyledModalInner>
+        </StyledModalOuter>
+      </>
+    )
+  );
+}
+
+const StyledModalOuter = styled.div`
+  overflow: none;
+`;
+
+const StyledModalInner = styled.div`
+  display: flex;
+  text-align: center;
+  flex-flow: column;
+  justify-content: space-between;
+  position: fixed;
+  z-index: 2;
+  left: 50%;
+  top: 50%;
+  bottom: 1px;
+  transform: translate(-50%, -50%);
+  width: 30vw;
+  /* min-width: 350px; */
+  height: 53vw;
+  background-color: rgba(24, 24, 24, 0.95);
+  color: white;
+  overflow: hidden;
+  overflow-y: scroll;
+  overflow-x: scroll;
+  scrollbar-width: thin;
+  position: absolute;
+  z-index: 999;
+  max-height: 1500px;
```

Comment:
I am a bit confused as you are using twice the z-index here as well as the position property 🤔 Please, have a look at this styled component again.

```suggestion
  display: flex;
  text-align: center;
  flex-flow: column;
  justify-content: space-between;
  position: fixed;
  z-index: 2;
  left: 50%;
  top: 50%;
  bottom: 1px;
  transform: translate(-50%, -50%);
  width: 30vw;
  height: 53vw;
  background-color: rgba(24, 24, 24, 0.95);
  color: white;
  overflow: hidden;
  overflow-y: scroll;
  overflow-x: scroll;
  scrollbar-width: thin;
  max-height: 1500px;
```